packages/deploy-script-support/README.md

```diff
@@ -15,93 +16,105 @@ it uses, identifies the "manifest", (which associates permissions to access
 powers in the bootstrap space with functions to be called), and builds bundles
 of source code needed by the proposal.
 
-`.../builders/scripts/vats/upgradeVaults.js` is a canonical example. It says the
-proposal to run is '@agoric/inter-protocol/src/proposals/upgrade-vaults.js',
-lists the manifest there as `'getManifestForUpgradeVaults'`, and directs the
-creation of a bundle from
-'@agoric/inter-protocol/src/vaultFactory/vaultFactory.js', which will be made
-available to the proposal as `vaultsRef` in options.
+Here's a simple example from `start-valueVow.js`
 
-`upgrade-vaults.js` defines `getManifestForUpgradeVaults()`, which returns a
-`manifest` that says `upgradeVaults()` should be executed, and specifies what
-powers it should have access to.
+```
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }) =>
+  harden({
+    // Somewhat unorthodox, source the exports from this builder module
+    sourceSpec: '@agoric/builders/scripts/testing/start-valueVow.js',
+    getManifestCall: [
+      'getManifestForValueVow',
+      {
+        valueVowRef: publishRef(
+          install('@agoric/zoe/src/contracts/valueVow.contract.js'),
+        ),
+      },
+    ],
+  });
+```
+
+The first element of `getManifestCall` is interpreted as the name of a proposal.
+The second element of `getManifestCall` produces the `options` argument passed
+to the proposal (`{ valueVowRef, ... }` in the example above). A common thing to
+want to pass in options is a reference to code to be installed on-chain. The
+`valueVowRef` example above shows how. `publishRef(install(<path>))` is built
+from sources in the sdk, and passed as a `bundleRef`, which contains a
+`bundleID` suitable for passing to Zoe (for contracts) or `vatAdminService` (for
+non-contract vat code).
+
+It says the proposal to run is
+'@agoric/builders/scripts/testing/start-valueVow.js', lists the manifest there
+as `'getManifestForValueVow'`, and directs the creation of a bundle from
+'@agoric/zoe/src/contracts/valueVow.contract.js', which will be made
+available to the proposal as `valueVowRef` in options.
+
+The manifest gives the permissions for accessing the promise space, and passes
+the `installations` to the proposalBuilder. Notice that `valueVowRef` from the
+proposalBuilder is passed to `getManifest`, which adds it to `installations`,
+with the name `valueVow`. The latter must match the well-known name in
+`consume.installation` and `agoricNames.installation`. The call to `restoreRef`
```

Comment:
"must match the well-known name" seems an odd phrasing. You have to use the same name in all the relevant places, but if you don't, the software just assumes you meant to do that. These core eval scripts _define_ the well-known names.